### PR TITLE
[test-only] Api-test for sharing personal space

### PR DESCRIPTION
**Pull request by ScharfViktor**
related #6238 
 https://github.com/owncloud/ocis/pull/6271 

added new test:
- User cannot create share for personal space to user/group
- User cannot create public link for personal space

deleted issue tags

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiSpacesShares-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/22142/53/1) failed. Further test are cancelled...

File: tests/acceptance/features/apiSpacesShares/shareSpaces.feature [hunk 1]
```diff
@@ -435,3 +435,30 @@ Feature: Share spaces
       | manager |
       | editor  |
       | viewer  |
+
+
+  Scenario Outline: user cannot share a personal to user
```
Comment (individual-it) [inline tests/acceptance/features/apiSpacesShares/shareSpaces.feature:440]:
```suggestion
  Scenario Outline: user cannot share the personal space to an other user
```

File: tests/acceptance/features/apiSpacesShares/shareSpaces.feature [hunk 2]
```diff
@@ -435,3 +435,30 @@ Feature: Share spaces
...
+    And the OCS status message should be "can not add members to personal spaces"
+    And the user "Bob" should not have a space called "Brian Murphy"
+    Examples:
+      | role        |
+      | Space Admin |
+      | Admin       |
+      | User        |
+
+
+  Scenario: user cannot share a personal to group
```
Comment (individual-it) [inline tests/acceptance/features/apiSpacesShares/shareSpaces.feature:455]:
```suggestion
  Scenario: user cannot share the personal space to a group
```

File: tests/acceptance/features/apiSpacesShares/shareSpacesViaLink.feature [hunk 3]
```diff
@@ -106,3 +106,11 @@ Feature: Share spaces via link
     And the OCS status code should be "404"
     And the OCS status message should be "Wrong path, file/folder doesn't exist"
     And for user "Alice" the space "share space" should not contain the last created public link
+
+
+  Scenario: user cannot share a personal space to public via link
```
Comment (individual-it) [inline tests/acceptance/features/apiSpacesShares/shareSpacesViaLink.feature:111]:
```suggestion
  Scenario: user cannot create a public link from the personal space
```

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**